src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -20,6 +25,9 @@ export const MyFieldAndBackground = () => {
   const user = useSelector((store: RootState) => store.user);
   const personId = user?.cristinId ?? '';
 
+  const [searchTerm, setSearchTerm] = useState('');
```

Comment:
flere felter her, så kanskje greit å være eksplisitt på hvilken search term dette er?
```suggestion
  const [keywordSearchTerm, setKeywordSearchTerm] = useState('');
```